charts/linkerd-control-plane/values.yaml

```diff
@@ -118,6 +118,12 @@ policyController:
       # -- Amount of ephemeral storage that the policy controller requests
       request: ""
 
+  # -- Liveness and readiness probe timeouts for policy controller
+  #livenessProbe:
+  #  timeoutSeconds: 5
+  #readinessProbe:
+  #  timeoutSeconds: 5
+
```

Comment:
I think it's cleaner to leave these uncommented, while leaving their original defaults (1). Also these settings speak for themselves, so no need to add a doc comment.

Reply:
Modified.